packages/remix-dev/vite/build.ts

```diff
@@ -270,23 +303,42 @@ export async function build(
     });
   }
 
-  // Since we're potentially running multiple Vite server builds with different
-  // output directories, we need to clean the root server build directory
-  // ourselves rather than relying on Vite to do it, otherwise you can end up
-  // with stale server bundle directories in your build output
-  await cleanServerBuildDirectory(viteConfig, ctx);
+  await cleanBuildDirectory(viteConfig, ctx);
```

Comment:
Now that manifest files are written to the root of the build directory, the comment that was written here previously now applies more generally to any Remix build since it may result in a different set of manifest files.